code/game/objects/items/devices/battle_royale.dm

```diff
@@ -0,0 +1,117 @@
+/// Quietly implants people with battle royale implants
+/obj/item/royale_implanter
+	name = "royale implanter"
+	desc = "Subtly implants people with rumble royale implants, \
+		preparing them to struggle for their life for the enjoyment of the Syndicate's paying audience. \
+		Implants may cause irritation at site of implantation."
+	icon = 'icons/obj/medical/syringe.dmi'
+	icon_state = "nanite_hypo"
+	w_class = WEIGHT_CLASS_SMALL
+	/// Do we have a linked remote? Just to prevent headdesk moments
+	var/linked = FALSE
+
+/obj/item/royale_implanter/interact_with_atom(atom/interacting_with, mob/living/user, list/modifiers)
+	if(!isliving(interacting_with))
+		if (!istype(interacting_with, /obj/item/royale_remote))
+			return NONE
+		var/obj/item/royale_remote/remote = interacting_with
+		remote.link_implanter(src, user)
+		return ITEM_INTERACT_SUCCESS
+	if (!linked)
+		balloon_alert(user, "no linked remote!")
+		return ITEM_INTERACT_BLOCKING
+	if (DOING_INTERACTION_WITH_TARGET(user, interacting_with))
+		balloon_alert(user, "busy!")
+		return ITEM_INTERACT_BLOCKING
+	var/mob/living/potential_winner = interacting_with
+	if (potential_winner.stat != CONSCIOUS)
+		balloon_alert(user, "target unconscious!")
+		return ITEM_INTERACT_BLOCKING
+	if (!potential_winner.mind)
+		balloon_alert(user, "target too boring!")
```

Comment:
i would specify non-sentient here just because there's definitely not boring targets that don't have minds